.travis.yml

```diff
@@ -27,20 +27,19 @@ env:
         # The following versions are the 'default' for tests, unless
         # overridden underneath. They are defined here in order to save having
         # to repeat them for all configurations.
-        - PYTHON_VERSION=3.6
+        - PYTHON_VERSION=3.7
         - NUMPY_VERSION=stable
         - ASTROPY_VERSION=stable
         - MAIN_CMD='python setup.py'
         - SETUP_CMD='test'
-        - PIP_DEPENDENCIES='requests mimeparse requests_mock flake8'
```

Comment:
Why get rid of flake8 here?  Is it installed by default in 3.7?

Reply:
ci-helpers installs it